storage/localstore/schema.go

```diff
@@ -24,6 +40,23 @@ const DbSchemaHalloween = "halloween"
 
 const DbSchemaSanctuary = "sanctuary"
 
+// the "diwali" migration simply renames the pullIndex in localstore
+const DbSchemaDiwali = "diwali"
+
+type migration struct {
+	name          string             //name of the schema
+	migrationFunc func(db *DB) error // the migration function that needs to be performed in order to get to the NEXT schema name
```

Comment:
Maybe to rename `migrationFunc` to just `func`, this type is already called migration.